go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -84,7 +98,7 @@ If the schema for the existing table does not match the schema for the new file,
 
 A mapping file can be used to map fields between the file being imported and the table being written to. This can be used when creating a new table, or updating or replacing an existing table.
 
-` + schcmds.MappingFileHelp +
+` + jsonInputFileHelp +
```

Comment:
Probably still helpful to include the mapping file info here, since you can use a mapping file and a JSON input file together (from what I understand at least).